src/lambda/utils.ts

```diff
@@ -46,6 +57,49 @@ export async function getSelectedLambdaNode(awsContext: AwsContext, element?: Fu
     throw new Error('No lambda found.')
 }
 
+export async function getCloudFormationsForRegion(regionCode: string,
+                                                  lambdaFunctions: FunctionNode[]): Promise<CloudFormationNode[]> {
+    const client = await ext.sdkClientBuilder.createAndConfigureSdkClient(
+        opts => new CloudFormation(opts),
+        undefined,
+        regionCode
+    )
+
+    return await listCloudFormations(client, lambdaFunctions)
+}
+
+export async function listCloudFormations(cloudFormation: CloudFormation,
+                                          lambdaFunctions: FunctionNode[]): Promise<CloudFormationNode[]> {
+    // TODO: this 'loading' message needs to go under each regional entry
+    // in the explorer, and be removed when that region's query completes
+    const status = vscode.window.setStatusBarMessage('Loading CloudFormations...')
```

Comment:
Localize this text

Reply:
I already fixed the "Loading Lambdas..." in another function